cmd/runtimetest/main.go

```diff
@@ -30,6 +32,13 @@ const PrGetNoNewPrivs = 39
 const specConfig = "config.json"
 
 var (
+	defaultFS = map[string]string{
+		"/proc":    "proc",
```

Comment:
we call it procfs but the real type in Linux is proc. I will submit a PR to fix it in runtime-spec